.gitignore

```diff
@@ -32,6 +32,8 @@ last-run-results.html
 !lighthouse-core/test/fixtures/artifacts/**/*.devtoolslog.json
 !lighthouse-core/test/fixtures/traces/**/*.trace.json
 !lighthouse-core/test/fixtures/traces/**/*.devtoolslog.json
+!lighthouse-core/test/fixtures/config-helpers/config/node_modules
+!lighthouse-core/test/fixtures/config-helpers/node_modules
```

Comment:
```suggestion
!lighthouse-core/test/fixtures/config-helpers/**/*/node_modules
```

Reply:
Thanks for your suggestion!

Reply:
oh, oops, should just be `!lighthouse-core/test/fixtures/config/*/node_modules`

Reply:
Aah! It worked perfectly! Thanks a lot!